assets/src/containers/App.js

```diff
@@ -18,14 +22,24 @@ function App () {
 
   return (
     <Router basename='/test/courses/'>
-      <div>
-        <DashboardAppBar onMenuBarClick={setSideDrawerState} sideDrawerState={sideDrawerState} user={user} />
-        <SideDrawer toggleDrawer={setSideDrawerState} sideDrawerState={sideDrawerState} />
-        <Route path='/:courseId' exact component={IndexPage} />
-        <Route path='/:courseId/grades' component={GradeDistribution} />
-        <Route path='/:courseId/assignment' component={AssignmentPlanning} />
-        <Route path='/:courseId/files' component={FilesAccessed} />
-      </div>
+      <>
+        <DashboardAppBar
+          onMenuBarClick={setSideDrawerState}
+          sideDrawerState={sideDrawerState}
+          user={user}
+          courseName={loaded ? courseInfo.name : null}
+          courseId={courseId} />
+        <SideDrawer
+          toggleDrawer={setSideDrawerState}
+          sideDrawerState={sideDrawerState}
+          courseId={courseId}
+          courseInfo={loaded ? courseInfo : null} />
+        <Route path='/:courseId/' exact
+          render={props => <IndexPage {...props} courseInfo={loaded ? courseInfo : null} courseId={courseId} />} />
+        <Route path='/:courseId/grades' render={props => <GradeDistribution {...props} courseId={courseId} />} />
+        <Route path='/:courseId/assignment' render={props => <AssignmentPlanning {...props} courseId={courseId} />} />
+        <Route path='/:courseId/files' render={props => <FilesAccessed {...props} courseId={courseId} />} />
+      </>
     </Router>
```

Comment:
How is this commit relevant here? this PR is addressing removing HOC with hooks whereas this looks like calling hooks for getting courseInfo and some other changes based on that call.

isn't this supposed to be [PR#481](https://github.com/tl-its-umich-edu/my-learning-analytics/pull/481)

Reply:
Sorry, I think this was a mistake, I'll revert the `App.js` back to how it was.